pravegactl/README.md

```diff
@@ -0,0 +1,47 @@
+
+# Praveglet
```

Comment:
above it is called "Pravegactl"
These should be consistent.

Reply:
fixed it.